src/collections/members/bariq-nurlis/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name: Bariq Nurlis
+position: Contributor
+image_path: ./bariq-nurlis.jpeg
+github: bariqhibat
+linkedin: bariqhibat
+location: Singapore/Indonesia
+bio: Bariq is an undergraduate student who currently studying engineering in NTU Singapore. He is a passionate maker who likes to build things and watch them benefitting people's lives. He's excited to contribute and learn in the open source community.
+badges: meshery
```

Comment:
It should be like:
```
badges:
 - meshery
```
For example this [profile](https://raw.githubusercontent.com/layer5io/layer5/master/src/collections/members/aditi-ahuja/index.mdx)
@leecalcote

Reply:
```suggestion
badges: 
  - meshery
```